README.md

```diff
@@ -40,14 +49,33 @@ Consider using one of the other formats if that's the case.
 - id: files
   uses: Ana06/get-changed-files@v1.2
   with:
-    filter: '*.php'
+    glob-filter: '*.php'
 - run: |
     for changed_file in ${{ steps.files.outputs.all }}; do
       echo "Do something with this ${changed_file}."
     done
 ```
 
-## Get all added and modified files as CSV
+### Get all changed *.yml files but exclude .github/*/*.yml files
+
+Be careful that the order of the glob has an importance.
+Therefore, including all YML files first and excluding the YML files of your `.github/*/` directories is the way to go to exclude them.
+If those two globs were inverted, you **would** include all the YML files, with the ones in your `.github/*/` directories.
+
+```yaml
+- uses: Ana06/get-changed-files@v1.2
```

Comment:
I guess this example should be changed with the next version so that people are not confused?

Reply:
right!